src/types/global.ts

```diff
@@ -22,13 +22,37 @@ export type SchedulerRowLabel = {
   subtitle: string;
 };
 export type SchedulerProjectData = {
+  /**
+   * Unique Id of item
+   */
   id: string;
+  /**
+   * Represents start date of from which tile will render
+   */
   startDate: Date;
+  /**
+   * Represents end date to which tile will render
+   */
   endDate: Date;
+  /**
+   * Indicates how much time is spent per day. Given in seconds and converted by Scheduler to hours/minutes
+   */
   occupancy: number;
+  /**
+   * Title of item
+   */
   title: string;
+  /**
+   * Subtitle of item
+   */
   subtitle?: string;
+  /**
+   * Short description displayed on tile. Optional
```

Comment:
I don't think we need to specify that this is Optional, because it will be visible by type "string | undefined| or by '?'.
If still you want to add this, please add Optional to description, bgColor and subtitle